apps/designer/app/shared/db/misc.server.ts

```diff
@@ -1,5 +1,8 @@
+import { uploadAsset } from "@webstudio-is/asset-uploader";
 import * as db from "./";
 
+export const uploadAssets = uploadAsset;
```

Comment:
why reexporting though instead of using directly? maybe I miss something but I don't see why

Reply:
oh that's because remix makes sure to not load serverside code on the client, I think if you put the logic inside asset-uploader into a .server.ts file you won't need this

Reply:
but also tbh, maybe its better to do it this way, because a standalone package donesn't have to be concerned with remix specific stuff, but please name uploadAsset(s) consistently

Reply:
it also doesn't work lol

changed the names